tests/types.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Awaitable, Callable, Dict, Iterable
+
+from starlette.responses import Response
+from starlette.testclient import TestClient
+
+TestClientFactory = Callable[..., TestClient]
+
+WSGIResponse = Iterable[bytes]
+StartResponse = Callable[..., Any]
+Environment = Dict[str, Any]
+
+AsyncEndpoint = Callable[..., Awaitable[Response]]
+SyncEndpoint = Callable[..., Response]
```

Comment:
Can those types not be in `types.py`? They are only used once, right?

Reply:
Yes, that is right, I have moved them back in their files.